config/runtime.exs

```diff
@@ -229,12 +229,12 @@ case config_env() do
            Enum.join(
              [
                "default-src 'none'",
-               "img-src 'self' cdn.mbta.com #{System.get_env("STATIC_HOST", "")} #{System.get_env("CMS_API_BASE_URL", "")} *.googleapis.com *.gstatic.com *.s3.amazonaws.com data:",
+               "img-src 'self' cdn.mbta.com #{System.get_env("STATIC_HOST", "")} #{System.get_env("CMS_API_BASE_URL", "")} *.google.com *.googleapis.com *.gstatic.com *.s3.amazonaws.com data: i.ytimg.com www.googletagmanager.com",
                "style-src 'self' 'unsafe-inline' www.gstatic.com #{System.get_env("STATIC_HOST", "")}",
-               "script-src 'self' 'unsafe-eval' 'unsafe-inline' #{System.get_env("STATIC_HOST", "")} translate.google.com www.gstatic.com www.googletagmanager.com *.googleapis.com",
+               "script-src 'self' 'unsafe-eval' 'unsafe-inline' #{System.get_env("STATIC_HOST", "")} www.instagram.com *.google.com www.gstatic.com www.googletagmanager.com *.googleapis.com data.mbta.com",
                "font-src 'self' #{System.get_env("STATIC_HOST", "")}",
                "connect-src 'self' *.googleapis.com #{sentry_dsn_host || ""}",
```

Comment:
I believe `analytics.google.com` needs to be added here to `connect-src` too

Reply:
whoops looks like it's `www.google-analytics.com` (and needs to be in `script-src` as well as `connect-src`)